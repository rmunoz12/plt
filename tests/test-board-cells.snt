@setup {

group table(Rect(3, 3)) { };

}

@turns {

func void begin() {
    group table a = table();

    print(a.cells); print("\n");
    print(a.cells[2]); print("\n");
    end;
}

}
