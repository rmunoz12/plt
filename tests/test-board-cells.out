[<Group Piece instance>, <Group Piece instance>, <Group Piece instance>, <Group Piece instance>, <Group Piece instance>, <Group Piece instance>, <Group Piece instance>, <Group Piece instance>, <Group Piece instance>]
<Group Piece instance>
